api/SubtleCrypto.json

```diff
@@ -670,6 +670,55 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "ed25519": {
+          "__compat": {
+            "description": "<code>Ed25519</code> algorithm",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/SubtleCrypto/sign",
```

Comment:
This is a sub-feature of exportKey, why link to sign?

Reply:
copy-paste err, thanks for spotting